controllers/goharbor/harbor/ingresses.go

```diff
@@ -10,8 +10,9 @@ import (
 	"github.com/goharbor/harbor-operator/controllers/goharbor/notaryserver"
 	"github.com/goharbor/harbor-operator/pkg/graph"
 	"github.com/pkg/errors"
-	netv1 "k8s.io/api/networking/v1"
+	netv1beta1 "k8s.io/api/networking/v1beta1"
```

Comment:
This will cause too many conflicts with the develop branch.

What I'm suggesting is keeping `netv1` pointing to `k8s.io/api/networking/v1beta1` and import the `k8s.io/api/networking/v1` with another name like `netv1new`.

When later we remove the K8s old versions out of our supporting list and then we just need to modify the importing list and the `netv1new` packages which will be less.